src/Liquid.Core.Telemetry.ElasticApm/MediatR/LiquidPipelineBehavior.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Elastic.Apm.Api;
+using MediatR;
+using Microsoft.Extensions.Logging;
+
+namespace Liquid.Core.Telemetry.ElasticApm.MediatR
+{
+    /// <summary>
+    /// Implements <see cref="IPipelineBehavior<TRequest, TResponse>"/> for Elastic APM.
+    /// </summary>
+    /// <typeparam name="TRequest">The type of request.</typeparam>
+    /// <typeparam name="TResponse">Type of response object obtained upon return of request.</typeparam>
+    public sealed class LiquidPipelineBehavior<TRequest, TResponse> : IPipelineBehavior<TRequest, TResponse>
```

Comment:
The name of the class must represent what it is responsible for.
Example: "LiquidElasticApmTelemetryBehavior"